content/2_Bronze/Complete_Rec.mdx

```diff
@@ -974,19 +995,21 @@ int ans = 0;
  
 void dfs(int r) { // place queen in r-th row
 	if (r == 8) {
-		++ ans; // found valid placement
+		++ans; // found valid placement
 		return;
 	}
-	F0R(i,8) if (g[r][i] == '.' && !c[i] && !sum[i+r] && !dif[i-r+7]) {
+	for (int i = 0; i < 8; ++i) if (g[r][i] == '.' && !c[i] && !sum[i+r] && !dif[i-r+7]) {
 		c[i] = sum[i+r] = dif[i-r+7] = 1;
 		dfs(r+1);
 		c[i] = sum[i+r] = dif[i-r+7] = 0;
 	}
 }
  
 int main() {
-	setIO(); F0R(i,8) re(g[i]);
-	dfs(0); ps(ans);
+	setIO(); 
+	for (int i = 0; i < 8; ++i) cin >> g[i];
+	dfs(0); 
+	cout << ans;
 }
 ```
```

Comment:
Code looks good but in this case I think the code with templates is fine, and there is no need to change it. Nevertheless wait for another review.